src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -123,6 +151,6 @@ public OffsetDateTime offsetDateTime(LocalDateTime localTime) {
      * или сообщение "dateTime can't be formatted!"
      */
     public String formatDate(LocalDateTime dateTime) {
-        return "Date can't be formatted!";
+        return DateTimeFormatter.ofPattern("dd MMMM yyyy HH:mm").format(dateTime);
```

Comment:
let's make formatter a constant 
And you never return message "dateTime can't be formatted!" in this method, please complete this task according to description

Reply:
added